src/controllers/articleController.js

```diff
@@ -1,8 +1,68 @@
+import Article from '../models/article';
+import { ArticleServices } from '../services/articleServices';
+
 export class ArticleController {
-    // TODO Don't access database from this file you only needs
-    createArticle(req, res, next) { }
-    getAllArticles(req, res, next) { }
-    getArticle(req, res, next) { }
-    updateArticle(req, res, next) { }
-    deleteArticle(req, res, next) { }
+  // TODO Don't access database from this file you only needs
+  async createArticle(req, res, next) {
+    try {
+      const data = new Article({
+        title: req.body.title,
+        content: req.body.content,
+        image: req.file.path,
+      });
+      const article = await ArticleServices.createArticle(data);
+      res.send(article);
+    } catch (error) {
+      res.status(404);
+      res.send({ error: 'No article created' });
+    }
+  }
+  async getAllArticles(req, res, next) {
+    try {
+      const articles = await ArticleServices.getAllArticles();
+      res.send(articles);
+    } catch (error) {
+      res.status(404);
+      res.send({ error: 'Error! try again' });
```

Comment:
Change the status code. Refactor your code by `res.status(500).send({ error: 'Error! try again' })`. Implement this on other methods